packages/web-react/src/components/Spinner/README.md

```diff
@@ -0,0 +1,16 @@
+# Spinner
+
+Spinner allows users to show loading state.
```

Comment:
Too much copy-paste… 🙈 

How about reusing (a simplified version of) what you have in `web`?

> The Spinner indicates something is loading.